PayPal/src/test/java/com/braintreepayments/api/paypal/PayPalClientUnitTest.java

```diff
@@ -507,7 +514,43 @@ public void tokenize_whenPayPalInternalClientTokenizeResult_callsBackResult()
     }
 
     @Test
-<<<<<<< HEAD
+    public void edit_auth_payPalVaultEditRequest() {
```

Comment:
Lets rename these methods to match the examples already in this class. They all follow a similar pattern that these new tests do not follow. Seems like this one should be: `createEditAuthRequest_whenEditRequest_sendsPayPalEditRequestViaInternalClient`